grand_challenge_forge/partials/example-algorithm/example-algorithm{{cookiecutter._}}/Dockerfile

```diff
@@ -7,18 +7,18 @@ ENV PYTHONUNBUFFERED 1
 RUN groupadd -r user && useradd -m --no-log-init -r -g user user
 USER user
 
-WORKDIR /tmp
+WORKDIR /opt/app
```

Comment:
You'll need to create the directory first https://github.com/comic/evalutils/blob/33573e7a48d7f48fc6ff03f7ca78a227b13d292d/evalutils/templates/container/%7B%7B%20cookiecutter.package_name%20%7D%7D/Dockerfile#L3-L9

Reply:
Ah okay, Docker does this for you now. Nice.

Reply:
I trusted the tests =)